apps/docs/app/tokens.css

```diff
@@ -66,7 +66,8 @@
     --hd-color-white-surface: var(--hd-white);
     --hd-color-neutral-surface: var(--hd-neutral-20);
     --hd-color-neutral-surface-weak: var(--hd-neutral-50);
-    --hd-color-surface-neutral-selection: var(--hd-neutral-75);
+    --hd-color-surface-neutral-selection: rgb(64 86 117);
```

Comment:
seems weird to have an rgb color here. why is that?

Reply:
This could be an Hex, it's not a standardized token, the reason is that I picked the color using a color picker to "copy" another website.